### PR TITLE
feat: extract filter values to dashboard

### PR DESCRIPTION
**Pull request by shlev**
# Description
as requested in issue,  extracted filter values to Dashboard 
to apply changes in single place.

## screenshots
<paste here>

### CODE REVIEW

File: src/pages/dashboard/OperatorHbarChart/OperatorHbarChart.tsx [hunk 1]
```diff
@@ -13,17 +13,19 @@ const colorsByCompannies: { [index: string]: string } = {
   מטרופולין: '#FF8500',
 }
 
-const excludeOperators = [/רכבת ישראל/, /^מוניות.*/, /^ירושלים-.*/, /^ירושלים -.*/]
+// const excludeOperators = [/רכבת ישראל/, /^מוניות.*/, /^ירושלים-.*/, /^ירושלים -.*/]
```
Comment (NoamGaash) [inline src/pages/dashboard/OperatorHbarChart/OperatorHbarChart.tsx:16]: delete this line please

**Comment by NoamGaash** [comment]
please see the lint issues

File: src/pages/dashboard/OperatorHbarChart/OperatorHbarChart.tsx (hunk 1 above)
Comment (shlev) [reply src/pages/dashboard/OperatorHbarChart/OperatorHbarChart.tsx]: delete.
applied regex filtering of dashboard ByDay to ByHour
extracted reg to Dashboard component to apply 1 param on both childs.

**Review by NoamGaash — CHANGES_REQUESTED** [review]
could you please revert all the whitespace changes?

File: .vscode/launch.json
Comment (NoamGaash) [inline .vscode/launch.json:1]: please remove this file. you may add it to the gitignore

File: src/pages/dashboard/OperatorHbarChart/OperatorHbarChart.tsx [hunk 2]
```diff
@@ -35,10 +33,6 @@ function OperatorHbarChart({
           ...o,
           color: getColorName(o.name),
         }))
-        .filter((operator) => {
-          // Check if the string matches any of the patterns
-          return !excludeOperators.some((pattern) => pattern.test(operator.name))
-        })
```
Comment (NoamGaash) [inline src/pages/dashboard/OperatorHbarChart/OperatorHbarChart.tsx:41]: is it on purpose?

File: src/pages/dashboard/DashboardPage.tsx [hunk 3]
```diff
@@ -162,7 +173,10 @@ const DashboardPage = () => {
             {loadingGrap ? (
               <Skeleton active />
             ) : (
-              <ArrivalByTimeChart data={convertToGraphCompatibleStruct(graphData)} />
+              <ArrivalByTimeChart
+                data={convertToGraphCompatibleStruct(graphData)}
+                // excludeOperators={excludeOperators}
```
Comment (NoamGaash) [inline src/pages/dashboard/DashboardPage.tsx:178]: please remove commented out code

**Comment by NoamGaash** [comment]
you have conflicts - please merge the main branch

File: src/pages/dashboard/OperatorHbarChart/OperatorHbarChart.tsx (hunk 2 above)
Comment (shlev) [reply src/pages/dashboard/OperatorHbarChart/OperatorHbarChart.tsx:41]: yes

**Comment by ArkadiK94** [comment]
@shlev Hi, why did you closed this pr?

**Review by NoamGaash — APPROVED** [review]

**Pull request closed without merging**